src/index.js

```diff
@@ -1,10 +1,15 @@
-import Exchange from './component/exchange'
-import Buy from './component/buy'
-import Sell from './component/sell'
-import Send from './component/send'
-import Post from './Post'
 import './styles/styles.scss'
+import { CurrencyService } from './services'
+import { Store } from './store'
+import { CommonComponent, ResultComponent, CurrencySelectComponent } from './components'
+import { RequestBuilder } from './utils'
 
-const post = new Post('ToString')
+const store = new Store()
+const http = new RequestBuilder()
+const service = new CurrencyService(store, http)
+const currencyComponent = new CurrencySelectComponent(store, service)
+currencyComponent.renderElement()
+currencyComponent.renderOptions()
+const resultComponent = new ResultComponent(store, service)
 
-console.log('Post to string: ', post.toString())
+new CommonComponent('.exchange').setClick(resultComponent.exchange.bind(resultComponent))
```

Comment:
you should to use click event inside class because you need create elements into this class